### ISSUE TITLE
Configuring a Read-Write LDAP User Store doc issues

### ISSUE THREAD

**Issue by YasasRangika** [wso2/docs-apim#8439]
### Description

1. The information in the `Info` box is not very clear. When I first read the document, I got the impression that the APIM 4.4.0 pack comes with built-in LDAP support. Afterward, I discovered that APIM 4.4.0 pack does not have in-built LDAP support, and we need to have a WSO2 Identity Server product running to utilize LDAP. It's important to note that in-built LDAP support has been removed after Identity Server 5.11 version. For testing purposes, it's best to mention that to use LDAP, you just need to start IS-5.11 pack with port offset=1. Then, the provided configurations should also match (i.e. ConnectionURL="ldap://localhost:10390").

<img width="1076" alt="image" src="https://github.com/user-attachments/assets/27e76780-d645-479e-bf50-2e0ad47d58c2">

2. In Step 1 [1], configuring the settings in deployment.toml for IS LDAP can be confusing for first-time users.
```json
[user_store]
class="org.wso2.carbon.user.core.ldap.ReadWriteLDAPUserStoreManager"
type = "database"
```

If you are using the WSO2 Identity Server (WSO2IS) with LDAP, you should refer to the `user-mgt.xml` file in the `<WSO2IS_HOME>/repository/conf` directory to find the value for the `class` key. This time, the default value for the 'type' will be 'database_unique_id', which is included in the package.

3. The last point of the 5th step in [1] contains typing errors.

~~thefollowingcodesnipetinsteadofthe~~ > the following code snipet instead of the

[1] https://apim.docs.wso2.com/en/4.4.0/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store/#step-1-setting-up-the-read-write-ldap-user-store-manager

**Comment by RusJaI** [comment]
### Update [2024-10-31]
PR was added. 
But further identified that there are some ambiguous facts that need to be verified from IS team. Hence keeping this on-hold until getting their input

**Comment by RusJaI** [comment]
### Update [2024-11-01]
Merged the PR for 4.4.0. 
Note : Read-write LDAP documentation of IS 6.0.0 and 6.1.0 are identical at the moment. In 7.0.0 they've moved the properties tables into a new page while maintaining the same set of properties.  According to the compatibility matrix, APIM 4.4. is fully supported by IS 6.1.0 and 6.0.0. Hence adding the same documentation in IS 6.1.0 to APIM 4.4. and master branch.
<img width="1280" alt="Screenshot 2024-11-01 at 17 54 22" src="https://github.com/user-attachments/assets/e752a6dc-75b9-45cc-9e3a-e5a6e7a27c9b">

Added the PRs for other 4.x versions.

Please find the List of PRs : 
4.4.0 : https://github.com/wso2/docs-apim/pull/8601
4.3.0 : https://github.com/wso2/docs-apim/pull/8631
4.2.0: https://github.com/wso2/docs-apim/pull/8620

Master : https://github.com/wso2/docs-apim/pull/8630
